package.json

```diff
@@ -86,6 +86,7 @@
   },
   "jest": {
     "coverageDirectory": "./coverage/",
+    "testURL": "http://localhost",
```

Comment:
without specifying this specs blow up for me locally with: SecurityError: localStorage is not available for opaque origins